contracts/contracts/eosio.system/src/voting.cpp

```diff
@@ -18,26 +18,31 @@
 #include <cmath>
 
 namespace eosiosystem {
-   
    using eosio::const_mem_fun;
    using eosio::indexed_by;
    using eosio::singleton;
    using eosio::transaction;
+   using namespace std::literals::string_literals;
```

Comment:
It's better to use 
using namespace std::string_literals;

https://stackoverflow.com/questions/38950008/using-string-literals-without-using-namespace-std